plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -364,4 +368,17 @@ private static Optional<Location> getLocation(JsonProcessingException e)
     {
         return Optional.ofNullable(e.getLocation()).map(location -> new Location(location.getLineNr(), location.getColumnNr()));
     }
+
+    private static Optional<String> getComment(JsonNode node)
+    {
+        JsonNode metadata = node.get("metadata");
```

Comment:
An alternative to this method:

```
        return Optional.ofNullable(node.get("metadata"))
                .map(metadata-> metadata.get("comment"))
                .map(JsonNode::asText);
```

Not necessarily better, just another way of expressing (hopefully) the same functionality.